src/app/lib/utilities/ichefURL/index.js

```diff
@@ -2,28 +2,27 @@
 const WEBP_ORIGIN_CODES = ['cpsdevpb', 'cpsprodpb'];
 
 const buildPlaceholderSrc = (src, resolution) => {
-  const parts = src.split('/');
-  const [domain, media, imgService, width, ...extraParts] = parts;
-  const definedWidth = width.replace('$width', resolution);
-  const domainWithProtocol = `https://${domain}`;
-
+  // return urn schemes unmodified
+  if (src.includes('urn:')) return src;
+  const noProtocolSrc = src.replace('https://', '');
+  const urlParts = noProtocolSrc.split('/');
+  const [domain, mediaType, imgService, ...extraParts] = urlParts;
```

Comment:
maybe a better var name could be

```suggestion
  const [domain, mediaType, imgService, ...remainingUrlParts] = urlParts;
```

Reply:
I think you could actually remove the resolution part like this

```js
const [domain, mediaType, imgService, , ...remainingUrlParts] = urlParts;
```

Reply:
but what you have done with the slice may be more clearer 🤷